src/benchmarks/benchmarks.cpp

```diff
@@ -132,6 +132,30 @@ int main(int argc, char* argv[]) // NOLINT
                 rxcpp::observable<>::defer([] { return rxcpp::observable<>::iterate(std::array{123}, rxcpp::identity_immediate()); }).subscribe([](int v){ ankerl::nanobench::doNotOptimizeAway(v); });
             });
         }
+        SECTION("interval - interval + take(3) + subscribe + immediate")
+        {
+            TEST_RPP([&]()
+            {
+                rpp::source::interval(std::chrono::milliseconds(10), rpp::schedulers::immediate{}) | rpp::operators::take(3) | rpp::operators::subscribe([](size_t v) { ankerl::nanobench::doNotOptimizeAway(v); });
```

Comment:
can we have less delay dependent benchmark? for example, interval with 1 nanosecond ?

Reply:
done